net-wireless/gr-foo/gr-foo-9999.ebuild

```diff
@@ -1,45 +1,46 @@
-# Copyright 1999-2016 Gentoo Foundation
+# Copyright 1999-2019 Gentoo Authors
 # Distributed under the terms of the GNU General Public License v2
 
-EAPI=5
-PYTHON_COMPAT=( python2_7 )
-
-inherit cmake-utils python-single-r1
+EAPI=7
 
-DESCRIPTION="Some GNU Radio blocks that bastianbl uses"
-HOMEPAGE="https://github.com/rftap/gr-rftap"
+PYTHON_COMPAT=( python2_7 )
 
 if [[ ${PV} == 9999* ]]; then
 	inherit git-r3
 	EGIT_REPO_URI="https://github.com/bastibl/gr-foo.git"
+else
 	KEYWORDS=""
-#else
-#	SRC_URI=""
-#	KEYWORDS=""
 fi
+inherit cmake-utils python-single-r1
```

Comment:
it is my understanding that inherit is supposed to be as close to the top as possible

Reply:
I think it's fine, with `git-r3` there's an inherit line very close to the top anyway and the whole block belongs together.